hack/templates/00-hypershift-dataplane-metrics-forwarder.SelectorSyncSet.yaml.tmpl

```
@@ -304,105 +304,129 @@ objects:
-                             tolerations:
-                               - effect: NoSchedule
-                                 key: node-role.kubernetes.io/infra
-                                 operator: Exists
-                             volumeClaimTemplate:
-                               metadata:
-                                 name: alertmanager-data
-                               spec:
-                                 resources:
-                                   requests:
-                                     storage: 10Gi
-                           telemeterClient:
-                             nodeSelector:
-                               node-role.kubernetes.io/worker: ""
-                             tolerations:
-                               - effect: NoSchedule
-                                 key: node-role.kubernetes.io/infra
-                                 operator: Exists
-                             telemeterServerURL: ${TELEMETER_SERVER_URL}
-                           prometheusOperator:
-                             nodeSelector:
-                               node-role.kubernetes.io/worker: ""
-                             tolerations:
-                               - effect: NoSchedule
-                                 key: node-role.kubernetes.io/infra
-                                 operator: Exists
-                           k8sPrometheusAdapter:
-                             nodeSelector:
-                               node-role.kubernetes.io/worker: ""
-                             tolerations:
-                               - effect: NoSchedule
-                                 key: node-role.kubernetes.io/infra
-                                 operator: Exists
-                           kubeStateMetrics:
-                             nodeSelector:
-                               node-role.kubernetes.io/worker: ""
-                             tolerations:
-                               - effect: NoSchedule
-                                 key: node-role.kubernetes.io/infra
-                                 operator: Exists
-                           openshiftStateMetrics:
-                             nodeSelector:
-                               node-role.kubernetes.io/worker: ""
-                             tolerations:
-                               - effect: NoSchedule
-                                 key: node-role.kubernetes.io/infra
-                                 operator: Exists
-                           thanosQuerier:
-                             nodeSelector:
-                               node-role.kubernetes.io/worker: ""
-                             tolerations:
-                               - effect: NoSchedule
-                                 key: node-role.kubernetes.io/infra
-                                 operator: Exists
+                                   topologyKey: topology.kubernetes.io/zone
```

```suggestion
                                  topologyKey: topology.kubernetes.io/zone
```
And following lines...